modules/client-configuration/pages/registration-bootstrap.adoc

```diff
@@ -29,15 +21,15 @@ When you create your bootstrap script, add a path to the file containing the pub
 ====
 SLES 15 uses Python 3 by default.
 Bootstrap scripts based on Python 2 must be re-created for SLES 15 systems.
-Attempting to register SLES 15 systems with {susemgr} using Python 2 bootstrap scripts will fail.
+Attempting to register SLES 15 systems with using Python 2 bootstrap scripts will fail.
 ====
 
 This procedure describes how to generate a bootstrap script.
 
 [[create.boot.script]]
 .Procedure: Creating a Bootstrap Script
-. From the {productname} {webui}, browse to menu:Main Menu[Admin > Manager Configuration > Bootstrap Script]. For more information, see pass:c[xref:FILENAME.adoc#s3-sattools-config-bootstrap[]].
-. In the [guimenu]``SUSE Manager Configuration - Bootstrap`` dialog disable [guimenu]``Bootstrap using Salt``.
+. In the {productname} {webui}, navigate to menu:Admin[Manager Configuration > Bootstrap Script].
+. In the [guimenu]``SUSE Manager Configuration - Bootstrap`` dialog, disable [guimenu]``Bootstrap using Salt``.
```

Comment:
I think we should mention both ways, disabling `Bootstrap using Salt` is necessary only when registering as traditional client.